models/balancer/ethereum/balancer_ethereum_sources.yml

```
@@ -1,25 +1,51 @@
 version: 2
 
 sources:
+  - name: balancer_v2_ethereum
+    description: >
+      Decoded tables related to Balancer v2, an automated portfolio manager and trading platform.
+    tables:
+      - name: Vault_evt_PoolRegistered
+        description: >
+          Decoded table of registered pools on the Balancer Vault contract.
+        loaded_at_field: evt_block_time
+        columns:
+          - &contract_address
+            name: contract_address
+            description: 'Ethereum address for the Balancer Vault contract used in transaction'
```

```suggestion
            description: 'Ethereum address for the Balancer Vault contract'
```

"used in the transaction" sounds to me that the Vault address could change.

---

Fixed the description.